tests/test_integration/test_code_generation.py

```diff
@@ -527,26 +544,23 @@ def k_to_ijk(outp: Field[np.float64], inp: Field[gtscript.K, np.float64]):
     data = np.arange(10, dtype=np.float64)
     inp = gt_storage.from_array(
         data=data,
-        shape=(10,),
-        default_origin=(0,),
+        aligned_index=(0,),
         dtype=np.float64,
-        mask=[False, False, True],
+        dimensions="K",
```

Comment:
Should this be `dimensions=["K"],`?

Reply:
A string is also a sequence of strings, which means either is fine, (unless there are >10 data dimensions).